pkg/util/httprequest/httprequest.go

```diff
@@ -68,14 +66,6 @@ func SchemeHost(req *http.Request) (string /*scheme*/, string /*host*/) {
 
 	forwardedProto := forwarded("Proto")
 	forwardedHost := forwarded("Host")
-	// If both X-Forwarded-Host and X-Forwarded-Port are sent, use the explicit port info
-	if forwardedPort := forwarded("Port"); len(forwardedHost) > 0 && len(forwardedPort) > 0 {
```

Comment:
As X-Forwarded-Host has no standard at all I use [RFC 7239](https://tools.ietf.org/html/rfc7239). It says that Forwarded host should have `the original value of the "Host" header field`. So if we have this header, then we know. Do you know any real reverse proxy that requires this code?